easybuild/easyconfigs/e/ELPA/ELPA-2021.11.001-intel-2022b.eb

```diff
@@ -0,0 +1,47 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Authors::   Inge Gutheil <i.gutheil@fz-juelich.de>, Alan O'Cais <a.ocais@fz-juelich.de>
+# License::   MIT/GPL
+#
+##
+
+name = 'ELPA'
+version = '2021.11.001'
+
+homepage = 'https://elpa.rzg.mpg.de'
```

Comment:
there's a new homepage since rzg renamed to mpcdf a few years ago -- `s/rzg/mpcdf/`
See also: https://github.com/easybuilders/easybuild-easyconfigs/pull/17358

Reply:
many thanks for spotting this error.